committeeoversightapp/templates/hearing_detail.html

```diff
@@ -19,14 +19,11 @@ <h2 class="text-center">{{hearing.name|title }}</h2>
          <td><b>Committees</b></td>
          <td>
            {% for committee in committees %}
-             {% if committee.is_subcommittee %}
-                <a href="{{ committee.parent_url }}">
-                  {{committee.parent}}</a>, {{committee}}
-              {% else %}
-                <a href="{{ committee.url }}">{{committee}}</a>
-             {% endif %}
+             {% autoescape off %}
+              {{ committee.get_linked_html }}
```

Comment:
Is this the same as `{{ variable|safe }}`?

Reply:
Yep https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#safe